packages/tools/pie-wrapper-react/scripts/add-react-wrapper.js

```diff
@@ -102,21 +102,27 @@ export function addReactWrapper (customElementsObject, folderName = process.argv
                 eventsObject = `{\n${eventNames}    }`;
             }
 
+            const componentPropsExportName = `${component.class.name.replace(/^Pie/, '')}Props`;
+
             // Create the main source code
-            componentSrc = `
-import * as React from 'react';
+            componentSrc = `import * as React from 'react';
 import { createComponent${component.class.events?.length > 0 ? ', EventName' : ''} } from '@lit/react';
-import { ${component.class.name} as ${component.class.name}React } from './index';
+import { ${component.class.name} as ${component.class.name}Lit } from './index';
+import { ${componentPropsExportName} } from './defs';
 
 export * from './defs';
 
-export const ${component.class.name} = createComponent({
+const ${component.class.name}React = createComponent({
```

Comment:
LGTM! small comment - you may need to update the example in the README so it matches this format 😄